### ISSUE TITLE
Crashes

### ISSUE THREAD

**Issue by NomarGomez** [pepijnverburg/osrs-runelite-twitch-live-loadout-plugin#56]
The plugin makes the client crash on the first phase of The Maiden of Sugadinti.

**Comment by pepijnverburg** [comment]
Hi there, thanks for the report. In the previous update I have improved stability when dealing with combat situations, but still it looks like there are some issues.

Could you get some of the error logs from your RuneLite client and post them here? From the RuneLite WIKI:
```
To find logs, do one of the following things:

- Open the screenshot directory (if you have access to client by right-clicking "Camera" button) and navigate 1 directory up and then open logs folder

- Navigate to %userprofile%\.runelite\logs on Windows or $HOME/.runelite/logs on Linux and macOS.
```

In these logs do you see any mentions of `Exception` or `Error:`? If possible could you share them with me?

**Comment by pepijnverburg** [comment]
This issue has been resolved in v1.0.2. Thanks for the report.
